pylint/reporters/json_reporter.py

```diff
@@ -48,6 +60,62 @@ def display_reports(self, layout: Section) -> None:
     def _display(self, layout: Section) -> None:
         """Do nothing."""
 
+    @staticmethod
+    def serialize(message: Message) -> Any:
+        raise NotImplementedError
+
+    @staticmethod
+    def deserialize(message_as_json: Any) -> Message:
```

Comment:
```suggestion
    def deserialize(message_as_json: OldJsonExport) -> Message:
```

Reply:
It will break if we return something different for the New Json Reporter. Let's discuss this in the follow-up PR.

Reply:
Yeah, I understand. But that's why I think it should be added here now and then changed in the follow-up. To keep this PR self-contained this method should expect a `OldJsonExport`. In a follow-up we can then make it handle more different types.